src/utils/helpers.ts

```diff
@@ -72,6 +72,18 @@ export const isBrowser = typeof window !== 'undefined' && typeof window.document
 
 export const noop = (): void => {};
 
+export const isEmpty = (value?: string | number | readonly string[] | undefined): boolean => {
+  if (value === null || value === undefined) {
+    return true;
+  }
+
+  if (typeof value === 'string' && value === '') {
+    return true;
+  }
+
```

Comment:
can we also add:

```ts
if (Array.isArray(value) && value.length === 0) {
  return true
}

if (typeof value === 'object' && Object.keys(value).length === 0) {
  return true
}
```

and update tests accordingly?

Reply:
Yeap, nice catch. Updated.